backend/endpoints/rooms.py

```diff
@@ -61,3 +63,87 @@ async def create_room(
         status_code=status.HTTP_424_FAILED_DEPENDENCY,
         detail="unable to create room",
     )
+
+@router.delete("/org/{org_id}/rooms/{room_id}/members/{member_id}",
+    response_model=ResponseModel, )
+async def remove_member(org_id: str, member_id: str, room_id: str, mem_id: str):
+    """Removes a member from a room.
+
+    Fetches the room which the member is removed from from the database collection
+    Pops the member being removed from the room's members dict
+    Updates the database collection with the new room
+    Returns the room dict if member was removed successfully
+
+    Args:
+        org_id (str): A unique identifier of an organisation
+        request: A pydantic schema that defines the room request parameters
+        member_id (str): A unique identifier of the member removing another member
+        room_id (str): A unique identifier of the room a member is being removed from
+        memb_id (str): A unique identifier of the member being removed from the room
+
+    Returns:
+        HTTP_200_OK (member removed from room): {room}
+    Raises
+        HTTP_404_NOT_FOUND: room or member not found
+        HTTP_403_FORBIDDEN: not authorized to remove room 
+        HTTP_424_FAILED_DEPENDENCY: member removal unsuccessful
+    """
+
+    room_obj = await get_room(org_id, room_id)
+
+    if not room_obj:
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail="room does not exist",
+        )
+    
+    if room_obj["room_type"] != RoomType.CHANNEL:
+        raise HTTPException(
+            status_code=status.HTTP_403_FORBIDDEN,
+            detail="unable to remove room member",
+        )
+    
+    if member_id not in room_obj["room_members"]:
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail="user not a member of the room",
+        )
+    
+    if member_id == mem_id:
+        raise HTTPException(
+            status_code=status.HTTP_403_FORBIDDEN,
+            detail="unable to remove room member",
+        )
+
+    member: RoomMember = room_obj["room_members"].get(member_id)
+    
+    if member.role != Role.ADMIN:
+        raise HTTPException(
+            status_code=status.HTTP_403_FORBIDDEN,
+            detail="unable to remove room member",
+        )
+
+    remove_member = room_obj["room_members"].pop(mem_id, "not_found")
+    
+    if remove_member != "not_found":
+        update_room = DB.update(ROOM_COLLECTION, room_obj["id"], room_obj)
+        if update_room == None:
```

Comment:
Here's a better way, when comparing to None
```suggestion
        if update_room is None:
```